pennylane/optimize/qng.py

```diff
@@ -104,7 +113,7 @@ class QNGOptimizer(GradientDescentOptimizer):
     ...     return qml.expval(qml.X(0) + qml.X(1))
 
     Once constructed, the cost function can be passed directly to the
-    optimizer's ``step`` function:
+    optimizer's :meth:`~.step` function:
```

Comment:
Not something that can/should be resolved in this PR, but I just found this weird Sphinx bug where the reference to `step` only works if the "Methods" drop down list in the rendered docs is expanded, otherwise nothing changes. With the list minimized, it only goes to the correct `step` docs when I refresh the page. Check it out for yourself [here](https://xanaduai-pennylane--5926.com.readthedocs.build/en/5926/code/api/pennylane.QNGOptimizer.html)

Reply:
That's funky, yeah!

Reply:
Resolving because it is tracked/discussed internally. Thanks for catching this!